### ISSUE TITLE
critical bug in savings account transaction limit

### ISSUE THREAD

**Issue by justkelvin** [justkelvin/finance#1]
A bug causes a savings account to withdraw cash when the limit is not reached. its weird logic bug let me explain clearly
- A user creates a savings account and enters daily amount to transact e.g. 5000
- maximum amount to withdraw e.g. 2000
- He then withdraws 2000 two times. Now he should only be able to withdraw 1000 because his limit is 5000, but when he withdraws 2000, which satisfies 
his max withdraw limit, he is able to bypass the 5000 daily spend limit somehow withdrawing 6000.
- Basically, if he now tries to withdraw he is unable to because the check somehow prevents him to.

**Comment by justkelvin** [comment]
Not fixed, sorry 😔😐
